### PR TITLE
Please help with buttons

### PR DESCRIPTION
**Pull request by SerhiiUnhurian**
- [DEMO LINK](https://SerhiiUnhurian.github.io/js_2048_game/)

### CODE REVIEW

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Demo link, you need to make a deploy
image

**Comment by SerhiiUnhurian** [comment]
deployed

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Good work, but lets make some improvement 
1. dont show cells before start 
2. i can click in one direction all time even when shouldn't
3. havent any lose message
image
 4.restart button doesn't work

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Not fixed previous comments or ypur forgot to redeploy work
2023-10-18_16h09_47

**Comment by SerhiiUnhurian** [comment]
please try it now

File: src/index.html
Comment (sTorba24) [inline src/index.html:23]: remove comments

File: src/scripts/main.js
Comment (sTorba24) [inline src/scripts/main.js:14]: remove comments

File: src/scripts/main.js
Comment (sTorba24) [inline src/scripts/main.js:145]: use switch case

File: src/styles/main.scss
Comment (sTorba24) [inline src/styles/main.scss:22]: remove all commented text

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Good work! lets fix mismatch with original game
when nothing to merge and no space to move cells game shouldn't continue
try to click only 'right' button for example and do the same in original game https://2048game.com/

File: src/index.html
Comment (SerhiiUnhurian) [reply src/index.html:23]: when nothing to merge and no space to move cells game shouldn't continue
try to click only 'right' button for example and do the same in original game https://2048game.com/

How can i do this?

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! Just couple tiny things needs to be fixed before approval)

1.return grid on bord 
image
2. i still can click just in right 
image

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Last mentor comment means that field should have visible cells 4 x 4
2023-10-19_15h47_55

Do not mutate function args
2023-10-19_15h49_11

**Comment by SerhiiUnhurian** [comment]
так я зрозумів, але воно працювало коли я робив завантаження при запуску
window.onload(setGame());
але сказали зробити с кнопки, то скажіть як мені зробити те розчерчення?

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Please ask mentors help in fe_chat on platform

**Comment by SerhiiUnhurian** [comment]
будь ласка перевірте мою структуру, я не знаю як можливо зробити розчерчення коли в мене 1 дів тільки і ви сказали додавати клітинки тільки при натисканні кнопки

**Review by denys-danyliuk — APPROVED** [review]

**Pull request closed without merging**
